pages/slack_guide.js

```diff
@@ -0,0 +1,33 @@
+import Head from 'components/head';
+import HeroBanner from 'components/HeroBanner/HeroBanner';
+import Content from 'components/Content/Content';
+import styles from 'styles/slack_guide.module.css';
+
+const pageTitle = "Slack Guide";
```

Comment:
You're hitting a lint error because " was used instead of '

Reply:
Thank you for the review, I will make these corrections.